src/main/java/core/basesyntax/figures/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figures;
+
+public interface AreaCalculator {
+    double getArea();
```

Comment:
Calculator -> calculate
```suggestion
    double calculateArea();
```